projects/core/src/lib/map/bookmark/bookmark.helper.ts

```diff
@@ -1,6 +1,11 @@
 import { MapSizeHelper, MapViewDetailsModel, MapUnitEnum } from '@tailormap-viewer/map';
-import { AppLayerModel } from '@tailormap-viewer/api';
 import { TristateBoolean, LayerVisibilityBookmarkFragment, LayerInformation } from './bookmark_pb';
+import { AppLayerWithInitialValuesModel } from '../models';
+
+export interface MapBookmarkContents {
+  visibilityChanges: { id: number; checked: boolean }[];
```

Comment:
Some nitpicking, but maybe rename id: number also to layerId: number to be consisten

Reply:
These are redefinitions of the parameters taken by the `setLayerVisibility` and `setLayerOpacity` actions, which do take `id` and `layerId` respectively. (this is also why it takes `checked` instead of e.g. `visible`.) I considered fixing both but it felt a bit much to refactor that for this PR.